loader/db.go

```diff
@@ -273,3 +291,215 @@ func isMySQLError(err error, code uint16) bool {
 	e, ok := err.(*mysql.MySQLError)
 	return ok && e.Number == code
 }
+
+func modifySQL(db *sql.DB, err error, sql string) (string, error) {
```

Comment:
can you provide an example, the origin sql and the modified sql?

Reply:
for example
source sql is
INSERT INTO `t3` VALUES
('aaa',',trx\',\'\'/13,.x','rrr',7), 
('bbb','qqq','ttt',6),
('123456','repqwd','66668888',2),
('ddd','eee','`~!@#?%&*()+',11);

if we find the 'repqwd' is a confilct value, we can get the pos is 2
after we delete it
modified sql is
INSERT INTO `t3` VALUES
('aaa',',trx\',\'\'/13,.x','rrr',7), 
('bbb','qqq','ttt',6),
('ddd','eee','`~!@#?%&*()+',11);

and duplicate data pair will be written into file like this
Key: b
Data: repqwd
Pos: 2
INSERT INTO t3 VALUES('123456','repqwd','66668888',2);

tips:if all data pair are duplication, we will get final SQL is 
INSERT INTO `t3` VALUES;
this is a wrong SQL, so we added a code to process it
else if isMySQLError(dupErr, tmysql.ErrParse) && dupErrFlag {
	ctx.L().Warn("execute statement", zap.String("Modified", "All data is dupError"))
	return nil // All data is dupError
}